app/src/main/res/values/strings.xml

```diff
@@ -715,4 +715,8 @@
     <string name="night_theme_summary">Select your favorite night theme — %s</string>
     <string name="select_night_theme_toast">You can select your favorite night theme below</string>
     <string name="download_has_started">Download has started</string>
+    <string name="cover_image">Cover Image</string>
+    <string name="download_cover_preference_title">Initially add cover image</string>
+    <string name="download_cover_preference">cover_pref_key</string>
```

Comment:
This particular line should go in `settings_keys.xml`, and you should set `translatable="false"` on it.